### DISCUSSION TITLE
hi all 🖐is this project still being updated?

### DISCUSSION THREAD

**Discussion by auna93** [quii/learn-go-with-tests#819] [Q&A] [answered]
I checked the commits of the project and the last one was many months ago, so i wanted to know if this project still being updated.

thanks a lot !

**Comment by quii** [comment, marked as answer]
What's the context? If I feel like there is a new chapter to write, and I have the time and space to do so, I'll do it. 

Ditto with the issues and pull requests. As I have time, I will deal with them. 

Is there new content you're looking for? 

To be frank, one of the delights of Go is it is backward compatible, so the content doesn't _need_ relentless updating.
